### ISSUE TITLE
[cli] bonobo run <path> / bonobo run -m <module>

### ISSUE THREAD

**Issue by hartym** [python-bonobo/bonobo#72]
```
$ bonobo run somedir
usage: bonobo run [-h] [--quiet] file
bonobo run: error: argument file: can't open 'somedir': [Errno 21] Is a directory: 'somedir'
```

Also, bonobo run -m .... should work.

**Comment by hartym** [comment]
Implemented in https://github.com/hartym/bonobo/tree/feature/cli_run

**Comment by hartym** [comment]
Merged in develop.

**Comment by hartym** [comment]
Solved in develop, target release: 0.4.0
